src/app/cli/src/init/client.ml

```diff
@@ -721,6 +721,23 @@ let set_staking =
                (Public_key.Compressed.to_base58_check
                   (Public_key.compress public_key)) ))
 
+let set_snark_work_fee =
+  Command.async ~summary:"Set fee reward for doing transaction snark work"
+  @@ Cli_lib.Background_daemon.init ~rest:true
+       Command.Param.(anon @@ ("fee" %: Cli_lib.Arg_type.txn_fee))
+       ~f:(fun port fee ->
+         let open Graphql_client in
+         let graphql =
+           Set_snark_work_fee.make
+             ~fee:(Yojson.Safe.to_basic @@ Currency.Fee.to_yojson fee)
```

Comment:
This makes me a little worried because I'm not sure what the yojson representation of Fee is. We want this to be a stringified int (or int64, wtv). In the past we've dealt with this by adding an "Encoders" module next to the "Decoders" one (in graphql_client.ml), and then using the functions in there to encode arguments to graphql queries/mutations